<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_OK">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>100</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK餐(任选4件)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_staples">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>主食</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_snacks">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小食</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sweety">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>甜点/配餐</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_drink">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>饮料</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_staples_spa">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>藤椒风味SPA堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_staples_yes">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>滋滋YES烤鸡腿堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_snacks_fries">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>薯条(中)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_snacks_chickenRice">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>劲爆鸡米花(小)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_sweety_redBeanPie">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>红豆派(1只)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_sweety_mashedPotato">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>醇香土豆泥</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_sweety_sundae">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>原味圣代</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_sweety_sundaeStrawberry">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>经典草莓圣代</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_drink_coke">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>可乐(冷/中)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_drink_juice">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>果汁(冷)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtn_drink_lemonTea">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>柠檬茶(冷/中)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
